locales/en/plugin__networking-console-plugin.json

```diff
@@ -52,6 +56,7 @@
   "Alternate Service target": "Alternate Service target",
   "Alternate Service to route to.": "Alternate Service to route to.",
   "Alternate Service weight": "Alternate Service weight",
+  "An error occurred": "An error occurred",
```

Comment:
Should this have a period at the end? There's already a translation entry for this text with a period at the end.